package.json

```diff
@@ -8,9 +8,11 @@
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.19.2",
     "bootstrap": "^4.4.1",
+    "extends-classes": "^1.0.5",
```

Comment:
I don't see this used anywhere so what is it? you can do npm uninstall extends-classes to get rid of it if it isn't used anywhere.